### ISSUE TITLE
Next steps with React

### ISSUE THREAD

**Issue by mxajPrice** [hackforla/access-the-data#246]
### Overview
Get an understanding of options for developing the front end for CKAN.  Seeing if React is the option we want to go with.

### Action Items
@nyarly to edit these action items.

Once this issue is done:

- [ ] revisit the topic of whether or not it makes sense to use the carousel design or if we should repurpose that design for page headings.  Ref: [18 Jan 2023 Team Meeting](https://github.com/hackforla/access-the-data/issues/63#issuecomment-1386173534)

**Comment by mxajPrice** [comment]
This is no longer needed.
